packages/block-editor/src/components/link-control/input-search.js

```diff
@@ -13,31 +13,31 @@ import { URLInput } from '../';
 
 const LinkControlInputSearch = ( {
 	value,
-    onChange,
-    onSelect,
-    renderSuggestions,
-    fetchSuggestions,
-    onReset,
-    onKeyDown,
-    onKeyPress,
+	onChange,
+	onSelect,
+	renderSuggestions,
+	fetchSuggestions,
+	onReset,
+	onKeyDown,
+	onKeyPress,
 } ) => {
 	return (
 		<form>
 			<URLInput
 				className="block-editor-link-control__search-input"
 				value={ value }
 				onChange={ onChange }
-				onKeyDown={ ( event, suggestion ) => {
+				__experimentalOnKeyDown={ ( event, suggestion ) => {
 					if ( event.keyCode === ENTER ) {
 						onSelect( suggestion )( event );
 					}
 					onKeyDown( event, suggestion );
 				} }
-				onKeyPress={ onKeyPress }
+				__experimentalOnKeyPress={ onKeyPress }
 				placeholder={ __( 'Search or type url' ) }
-				renderSuggestions={ renderSuggestions }
-				fetchLinkSuggestions={ fetchSuggestions }
-				handleURLSuggestions={ true }
+				__experimentalRenderSuggestions={ renderSuggestions }
+				__experimentalFetchLinkSuggestions={ fetchSuggestions }
```

Comment:
I think this should resolve https://github.com/WordPress/gutenberg/pull/17846#discussion_r339006840 for @talldan right?

Reply:
Yep, looks good 👍